jupyterlab_bigquery/src/components/list_items_panel/list_tree_item_widget.tsx

```diff
@@ -1,16 +1,10 @@
 import { UseSignal } from '@jupyterlab/apputils';
 import { Signal } from '@phosphor/signaling';
 import * as React from 'react';
-import { JupyterFrontEnd } from '@jupyterlab/application';
-import { ReduxReactWidget } from '../../utils/ReduxReactWidget';
+import { ReduxReactWidget } from '../../utils/widgetManager/redux_react_widget';
 import { ListProjectsService } from './service/list_items';
-import { WidgetManager } from '../../utils/widget_manager';
 import ListItemsPanel from './list_tree_panel';
-
-export interface Context {
-  app: JupyterFrontEnd;
-  manager: WidgetManager;
-}
```

Comment:
Duplicate with [this in `list_tree_panel`](https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/6c915a28bba6e6f571922de585b90fc39c973710/jupyterlab_bigquery/src/components/list_items_panel/list_tree_panel.tsx#L21-L23).  Deleting here and rerouted all usage to that in `list_tree_panel`